public/sw.js

```diff
@@ -43,16 +44,15 @@ self.addEventListener('activate', event => {
 self.addEventListener('fetch', event => {
   console.log(`fetch request for ${event.request.url}`)
   event.respondWith(
-    caches.match(event.request).then(cacheRes => {
-      return cacheRes ||
-        fetch(event.request)
-          .then((fetchResponse) => {
-            caches.open(staticCache).then(cache => {
-              // Must clone bc we can't put in cache && browser
-              cache.put(event.request, fetchResponse.clone());
-              return fetchResponse;
-          })
-        })
+    caches.match(event.request).then((response) => {
+      if (response) {
+        return response
+      }
+
+      return fetch(event.request).then((newResponse) => {
```

Comment:
Good job Ellie, we're getting a little closer. You may need to help me understand why exactly this code is solving some of our problems!